### DISCUSSION TITLE
Instructions on how to set up audio device

### DISCUSSION THREAD

**Discussion by tannenfreund87** [kraxarn/spotify-qt#198] [Q&A]
Thank you for this great app! It's so much faster and way less resource hungry than the "official" app or the spotify webapp and has all the functions I need!

Is it possible to write up a short instruction on how to set up an audio device for spotify-qt? I managed to install spotifyd and set it up with spotify-qt, but then spotify-qt is always asking for my spotify password. I don't see any option of storing it either in the global keychain or within the app itself. Setting up spotifyd with a *.conf file doesn't work for me, so I can set it up in a way it gets the access-token from spotify. And then I also read, it's possible to set it up with librespot? How would that work?

**Comment by souravsam0** [comment]
I'm also facing the same issue.
